tests/TWTezosAddressTests.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright © 2017-2019 Trust Wallet.
```

Comment:
can you create separate folder for `Tezos` in tests

Reply:
Done.